inference-engine/src/mkldnn_plugin/nodes/mkldnn_convert_node.cpp

```diff
@@ -0,0 +1,92 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "mkldnn_convert_node.h"
+#include "common/cpu_convert.h"
+
+using namespace mkldnn;
+using namespace MKLDNNPlugin;
+using namespace InferenceEngine;
+
+MKLDNNConvertNode::MKLDNNConvertNode(const InferenceEngine::CNNLayerPtr& layer, const mkldnn::engine& eng, MKLDNNWeightsSharing::Ptr &cache) :
+        MKLDNNNode(layer, eng, cache) {}
+
+void MKLDNNConvertNode::getSupportedDescriptors() {
+    if (outDims.empty() && output.getLayout() != InferenceEngine::Layout::ANY)
+        outDims.push_back(MKLDNNDims(output.getDims()));
+    if (inDims.empty() && input.getLayout() != InferenceEngine::Layout::ANY)
+        inDims.push_back(MKLDNNDims(input.getDims()));
+    if (getParentEdges().size() != 1)
+        THROW_IE_EXCEPTION << "Incorrect number of input edges for layer " << getName();
```

Comment:
Please refactor error messages as it was done for Split.

Reply:
Done in e1639f394ed3299b1def7fd04e3484aaeda0f5cd.